web/cobrands/fixmystreet.com/layout.scss

```diff
@@ -67,9 +67,21 @@ h3, h4,
 @import "_colours";
 @import "../sass/layout";
 
+body {
+    @media ($high-dpi-screen) {
+        background-image: url(images/tile@2x.jpg);
+        background-size: 500px;
+    }
+}
+
 // Tiled background stripe, not plain colour
 #front-main {
     background: $primary url(images/tile-y.jpg);
+
+    @media ($high-dpi-screen) {
+        background-image: url(images/tile-y@2x.jpg);
```

Comment:
Something is causing both `tile-y.jpg` and `tile-y@2x.jpg` to be requested, maybe because the non-retina version uses `background:` and the retina uses `background-image:`? (Or maybe because of the `border-image-source`, see other comment)